liquidhaskell-boot/src/Language/Haskell/Liquid/Transforms/CoreToLogic.hs

```diff
@@ -483,17 +483,32 @@ brels = M.fromList [ (symbol ("==" :: String), Eq)
                    , (symbol ("<" :: String) , Lt)
                    ]
 
+-- bops is a map between GHC function names/symbols and binary operators
+-- from the logic. We want GHC functions like +, -, etc. to bed mapped to the
```

Comment:
```suggestion
-- from the logic. We want GHC functions like +, -, etc. to map to the
```

Reply:
Actually I meant to "be mapped" but I can use "map" too